src/__tests__/components/deleteArticleButton/__test__/deleteArticleButton.test.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { DeleteButton, mapStateToProps } from "../../../../components/article/deleteArticleButton";
+
+const props = {
+  isArticleDeleted: false,
+  slug: "article-slug",
+  history: {
+    push: jest.fn(),
+  },
+  DeleteArticle: jest.fn(),
+};
+
+global.confirm = () => true;
```

Comment:
does this use the default `js` confirm?

Reply:
Yes, it uses the js alert box. Here I set it to true so meaning that in this case I basically click the ok button in the alert box. so it simulates that action for me in the test